google-cloud-tools-plugin/resources/META-INF/plugin.xml

```diff
@@ -177,11 +177,14 @@
             implementation="com.google.cloud.tools.intellij.appengine.cloud.AppEngineArtifactDeploymentSourceType"/>
 
     <framework.type implementation="com.google.cloud.tools.intellij.appengine.facet.AppEngineFrameworkType"/>
+    <framework.type implementation="com.google.cloud.tools.intellij.stackdriver.facet.StackdriverFrameworkType"/>
     <fileTemplateGroup implementation="com.google.cloud.tools.intellij.appengine.facet.AppEngineTemplateGroupDescriptorFactory"/>
     <facetType implementation="com.google.cloud.tools.intellij.appengine.facet.AppEngineStandardFacetType"/>
+    <facetType implementation="com.google.cloud.tools.intellij.stackdriver.facet.StackdriverFacetType"/>
     <project.converterProvider implementation="com.google.cloud.tools.intellij.appengine.migration.AppEngineStandardFacetConverterProvider"/>
     <project.converterProvider implementation="com.google.cloud.tools.intellij.appengine.migration.AppEngineStandardFacetMigrationConverterProvider"/>
     <framework.detector implementation="com.google.cloud.tools.intellij.appengine.facet.AppEngineFrameworkDetector"/>
+    <compileServer.plugin classpath="google-cloud-tools-jps-plugin.jar;appengine-plugins-core-0.2.0.jar;"/>
```

Comment:
Another option would be to omit the version number from appengine-plugins-core jar name.
I'll also open a new thread on IJ's support board asking if there is any way to script this.